### PR TITLE
ASC-429/ketcher-tool-integration

### PR DESCRIPTION
**Pull request by ShobhitExcelra**

### CODE REVIEW

File: apps/api/.eslintrc.js
Comment (himanshubexcelra) [inline apps/api/.eslintrc.js:1]: This file is not needed.

File: apps/api/prisma/schema.prisma
Comment (himanshubexcelra) [inline apps/api/prisma/schema.prisma:1]: This file is not needed.

File: apps/web/.eslintrc.js
Comment (himanshubexcelra) [inline apps/web/.eslintrc.js:1]: This file is not needed.

File: apps/web/app/ketchertool/components/Panel.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+  const getStructure = async () => {
+    KetcherFunctions.exportCtab().then((str) => {
+      const message = 'Export content:' + str;
+      console.log(message);
+      printToTerminal(message);
+    });
+
+    const url = process.env.REACT_APP_API_PATH!;
+    const response = await axios.post(url + '/indigo/aromatize', {
```
Comment (himanshubexcelra) [inline apps/web/app/ketchertool/components/Panel.tsx:62]: Please change it to fetch method. We are not using axios

File: apps/web/package.json [hunk 2]
```diff
@@ -11,14 +11,20 @@
     "test:watch": "jest --watch"
   },
   "dependencies": {
+    "axios": "^1.7.7",
```
Comment (himanshubexcelra) [inline apps/web/package.json:14]: You can remove axios. We are not using it

File: apps/web/package.json [hunk 3]
```diff
@@ -11,14 +11,20 @@
...
+    "ketcher-core": "^2.25.0",
+    "ketcher-react": "^2.25.0",
+    "ketcher-standalone": "^2.20.0",
     "next": "14.2.13",
     "react": "^18",
     "react-dom": "^18",
     "react-hot-toast": "^2.4.1"
   },
   "devDependencies": {
+    "@emotion/styled": "^11.11.5",
```
Comment (himanshubexcelra) [inline apps/web/package.json:26]: these two packages you can remove.

File: apps/web/app/ketchertool/components/Panel.tsx (hunk 1 above)
Comment (ShobhitExcelra) [reply apps/web/app/ketchertool/components/Panel.tsx]: Done.
https://github.com/himanshubexcelra/syncoop/pull/1/commits/a80416a6c7c376bc9e1d2319afd76c7f843c1d09

File: apps/web/package.json (hunk 3 above)
Comment (ShobhitExcelra) [reply apps/web/package.json]: Removed the axios but @emotion/styled is being used in 6-8 files. This package for embedding the CSS. Need to discuss before removing these codes.

**Pull request merged**